backend/agents/data_store.py

```diff
@@ -0,0 +1,38 @@
+from .tool import ToolMetadata
+from .types import Parameter
+from .agent import Agent, AgentMetadata
+
+
+@ToolMetadata(
+    name="Get Database Data",
+    description="This tool gets data from the database",
+    parameters={
+        "merchant": Parameter(
+            type="string",
+            description="Merchant name eg. Amazon",
+        )
+    },
+)
+def get_db_data(merchant: str) -> str:
+    print(f"Getting data from database with merchant: {merchant}")
+    return "£6.99"

```

Comment:
I know this is a stub, but considering we may be `summing` these values, should we just return a raw number and suppress the currency sign?. I guess this also has a bearing on how we store the data in the graph.

Reply:
ooooo good question! Instinctively, I agree with you. But at the moment this method is a tool, which is defined as a callable method that returns a string. I feel like everything will need to end up back as a string at some point, the question being what thing should do the conversion back to a string? 

Or should I just change this to `"6.99"` for now?

Reply:
Great questions..... It's not clear in my head what is the best return type to allow the next step to take this value and process it successfully. Feels like we need some kind of model/structure.

I'm happy to just use `"6.99"` for now, although long-term we'd need to know the currency type.

One option is that we return a dictionary that holds `amount` and `currency_code` or encode those values into a JSON string?

Shall we discuss on Monday how we future-proof return types for agent functions?